x/concentrated-liquidity/client/cli/tx.go

```diff
@@ -82,7 +82,7 @@ func NewCreateIncentiveCmd() (*osmocli.TxCliDesc, *types.MsgCreateIncentive) {
 	return &osmocli.TxCliDesc{
 		Use:                 "create-incentive [incentive-denom] [incentive-amount] [emission-rate] [start-time] [min-uptime]",
 		Short:               "create an incentive record to emit incentives (per second) to a given pool",
-		Example:             "create-incentive uosmo 69082 0.02 100 2023-03-03 03:20:35.419543805 24h --pool-id 1 --from val --chain-id osmosis-1",
+		Example:             "create-incentive uosmo 69082 0.02 2023-03-03 03:20:35.419543805 24h --pool-id 1 --from val --chain-id osmosis-1",
 		CustomFlagOverrides: poolIdFlagOverride,
```

Comment:
Can we create an issue to not require pool ID flag over ride and just accept it as an input param?

Reply:
sure, what's the main reasoning behind it tho? should we change it for `CreatePosition` too?

Reply:
We want to move away from required flags whenever possible (anything required should just be an input param).